source/whats-new.txt

```diff
@@ -55,6 +55,19 @@ What's New in 2.0
    :ref:`Connection Example Code <go-connection-example-code>` in the
    Connection Guide.
 
+The 2.0 {+driver-short+} release includes the following improvements and fixes:
+
+- Updates to monitoring event documents:
+
+  - The ``CommandStartedEvent`` and ``CommandFinishedEvent`` structs
```

Comment:
Q: Should this be `CommandFailedEvent` to match the change made above? Should it include `CommandSucceededEvent`?

Reply:
see my comment